plugins/osdn/common.go

```diff
@@ -214,10 +214,27 @@ func (oc *OvsController) StartNode(mtu uint) error {
 		return fmt.Errorf("Failed to start plugin: %v", err)
 	}
 
-	if oc.ready != nil {
-		close(oc.ready)
+	oc.MarkPodNetworkReady()
+
+	return nil
+}
+
+func (oc *OvsController) MarkPodNetworkReady() {
+	if oc.podNetworkReady != nil {
```

Comment:
It looks like we're always setting up oc.podNetworkReady, so we don't need to check for nil at all here.